### PR TITLE
Typography refactor: Break up huge _typography.scss file into smaller files

### PR DESCRIPTION
**Pull request by lyubomir-popov**
## Done

Move related functionality into separate files; remove unnecessary variables.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/

### CODE REVIEW

File: examples/templates/typographic-spacing.html [hunk 1]
```diff
@@ -538,43 +538,3 @@ <h6>Heading six</h6>
     <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam id venenatis turpis, ac tincidunt neque. Vestibulum dapibus sollicitudin tortor vel lacinia. Etiam et purus eu mi vehicula rhoncus non sit amet justo. Vestibulum quam elit, rhoncus a dolor ac, scelerisque aliquam tellus. Donec eget nisl odio. Suspendisse feugiat erat et.</p>
   </div>
 </div>
-
-  <div class="row" style="max-width: 100%">
```
Comment (lyubomir-popov) [inline examples/templates/typographic-spacing.html]: Removes the eyebrow example, as the styling will come in a separate PR

**Comment by kwm14** [comment]
## Design review
Looking good @lyubomir-popov approved Matrix as discussed but I just have one comment:

**vanilla-framework/examples/patterns/tables/table-mobile-card**
- [ ] spacing looks tight between rows on mobile view

screenshot 2019-01-30 at 17 07 46

**Comment by lyubomir-popov** [comment]
@kwm14 good catch. I'll fix it.

File: scss/_base_tables.scss [hunk 2]
```diff
@@ -29,7 +29,7 @@
   thead {
     th {
       @extend %muted-heading;
-      line-height: map-get($line-heights, table-headers);
+      line-height: map-get($line-heights, small);
```
Comment (deadlight) [inline scss/_base_tables.scss]: Nice generalisation and simplification :+1:

**Comment by kwm14** [comment]
Nice update @lyubomir-popov LGTM 👍

**Pull request merged**